go.mod

```diff
@@ -1,8 +1,8 @@
 module github.com/argoproj-labs/argocd-image-updater
 
-go 1.21
+go 1.22
 
-toolchain go1.22.3
+toolchain go1.22.4
 
```

Comment:
Upgrading go version is a major task on its own. I would prefer to do it as a separate issue later when the time is right.